src/scicookie/profile.py

```diff
@@ -13,8 +14,8 @@ class Profile:
     """Profile class that handles profiles defined in the .yaml files."""
 
     profile_name: str = ""
-    config: dict = {}  # noqa: RUF012
-    profiles_available: list = []  # noqa: RUF012
+    config: Dict[str, Any] = {}  # noqa: RUF012
+    profiles_available: List[str] = []  # noqa: RUF012
```

Comment:
Given that we support only Python 3.8+, this should ideally be -
```suggestion
    config: dict[str, Any] = {}  # noqa: RUF012
    profiles_available: list[str] = []  # noqa: RUF012
```

Reply:
I think that it just works for py3.9+

```
Python 3.8.1 | packaged by conda-forge | (default, Jan 29 2020, 14:55:04) 
[GCC 7.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> a: list[str] = []
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: 'type' object is not subscriptable
```

https://docs.python.org/3/library/typing.html#typing.Dict

Reply:
oh got your point, it works with `from __future__ import annotations` (that comment was below XD)
thank you so much, I didn't know that. I will push your suggestions! thanks!